client/internal/internal.go

```diff
@@ -0,0 +1,71 @@
+package internal
+import (
+    "github.com/derekparker/delve/proctl"
+    "github.com/derekparker/delve/goreadline"
+    "fmt"
+    "io"
+    "strings"
+    "os"
+    sys "golang.org/x/sys/unix"
+)
+const HistoryFile string = ".dbg_history"
```

Comment:
This should stay in `cli`, a websocket based frontend will not need access to the readline history file.